beard/similarity/pairs.py

```diff
@@ -273,3 +276,124 @@ def transform(self, X):
         X2 = X[:, n_features:]
 
         return np.abs(X1 - X2)
+
+
+class FuzzySimilarity(TransformerMixin):
+
+    """Implements Fuzzy Similarity (Soft TF-IDF) metric."""
+
+    def __init__(self, vocabulary, similarity, threshold):
+        """Initialize.
+
+        Parameters
+        ----------
+        :param vocabulary:
+            From word to feature_id or opposite.
+
+        :param similarity:
+            Similarity function which returns a number between 0 and 1.
+
+        :param threshold:
+            Threshold parameter that belongs to R.
```

Comment:
Ha, I assume you mean R=real numbers.

In that case, just simply document the parameter as follows

```
:param threshold: float
    ...
```

Please also describe what is this value thresholding.